lib/extensions/polyglot_with_database/coursemology/polyglot/language/python/python2_point7.rb

```diff
@@ -0,0 +1,7 @@
+module Extensions::PolyglotWithDatabase::Coursemology::Polyglot::Language::Python::Python2Point7
```

Comment:
What do you suggest?

It's an extension because the database functionality isn't used anywhere else.